tests/cupy_tests/core_tests/test_raw.py

```diff
@@ -1,7 +1,12 @@
+import os
 import pytest
+import shutil
+from tempfile import mkdtemp
```

Comment:
Please do not import non-module object. (as per OpenStack guideline)

```suggestion
import tempfile
```